.pipelines/templates/aks-cleanup.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - script: |
+      az login --identity
+      az group delete -n ${AZURE_CLUSTER_NAME} --yes --no-wait
+    displayName: Delete cluster and resource group
+    condition: always()
+    
```

Comment:
nit: Can you add a newline at the end of this file and the other files that are missing it?